static/app/views/relocation/relocation.spec.tsx

```diff
@@ -33,26 +72,84 @@ describe('Relocation', function () {
       expect(await screen.findByText('Choose a datacenter region')).toBeInTheDocument();
     });
 
-    it('should prevent user from going to the next step if no org slugs or region are entered', function () {
+    it('should prevent user from going to the next step if no org slugs or region are entered', async function () {
       renderPage('get-started');
```

Comment:
read the logic, it's called whenever any page is rendered.

Reply:
This API call is a bit slow when I tested it (not sure if this is just test region slowness or KMS storage or what). So I think we want to make this API call as early as possible, so that by the time the user gets to step 2, the key has already loaded.

However, since the user can technically refresh the page while on step 2, or because the API call can take a very long time in p95 cases (15 seconds +), we should have a nice UI to indicate that the key is still loading.